### PR TITLE
Update amagalma_ReaNoir track-item-take coloring utility.lua

### PR DESCRIPTION
**Pull request by amagalma**

### CODE REVIEW

File: Various/amagalma_ReaNoir track-item-take coloring utility.lua [hunk 1]
```diff
@@ -31,6 +31,11 @@
 
 --[[
  * Changelog:
+ * v.2beta (2017-04-04)
```
Comment (cfillion) [inline Various/amagalma_ReaNoir track-item-take coloring utility.lua]: You might want to fix this so users can see your nice changelog on update.

**Comment by cfillion** [comment]
I believe this is going to be the first script beta release shared through this repository! :+1:

File: Various/amagalma_ReaNoir track-item-take coloring utility.lua (hunk 1 above)
Comment (amagalma) [reply Various/amagalma_ReaNoir track-item-take coloring utility.lua]: Damn! I did it again!.. Fixing! :)

**Comment by amagalma** [comment]
It seems like the script does not get synchronized by ReaPack.. Is the "beta" in the version to blame?

**Comment by cfillion** [comment]
The logic of synchronize packages updates works like this:

-
  - If the option "Enable pre-release globally" is enabled: install the latest available version if the currently installed version number is lower
  - Else (default behavior): install the latest available *stable* version if the currently installed version number is lower
-  Install the latest available (any) version if the currently installed version number is a pre-release

This makes pre-releases "opt-in" until the next higher stable is released and without blocking the release of more 1.X stable releases in the meantime while the beta is getting worked on.

(Pre-releases are any version number that contains at least a letter).

**Pull request merged**